lib/travis/build/addons/homebrew.rb

```diff
@@ -0,0 +1,97 @@
+require 'travis/build/addons/base'
+require 'shellwords'
+
+module Travis
+  module Build
+    class Addons
+      class Homebrew < Base
+        SUPPORTED_OPERATING_SYSTEMS = %w[
+          osx
+        ].freeze
+
+        def before_prepare?
+          SUPPORTED_OPERATING_SYSTEMS.any? do |os_match|
+            data[:config][:os].to_s == os_match
+          end
+        end
+
+        def before_prepare
+          sh.fold('brew') do
+            update_homebrew if update_homebrew?
+            install_homebrew_packages
+          end
+        end
+
+        private
+
+        def config
+          @config ||= Hash(super)
+        end
+
+        def update_homebrew?
+          config[:update]
```

Comment:
This simply tests truthiness, not `true`. I suggest normalizing and comparing. For example:

```ruby
config[:update].to_s.upcase == 'true'
```